ArduCopter/switches.cpp

```diff
@@ -661,15 +661,20 @@ void Copter::do_aux_switch_function(int8_t ch_function, uint8_t ch_flag)
 
         case AUXSW_RC_OVERRIDE_ENABLE:
             // Allow or disallow RC_Override
-            switch (ch_flag) {
-                case AUX_SWITCH_HIGH: {
-                    ap.rc_override_enable = true;
-                    break;
-                }
-                case AUX_SWITCH_LOW: {
-                    ap.rc_override_enable = false;
-                    break;
+            // Prevent usage of this configuration on channel overridable. Enable RC_Override in case of radio lose.
```

Comment:
This is making a big assumption on what channels are overridable.

There are other options; a mask of which channels can be overridden, for example.  That mask would live next to where we pull apart the rc override packets, which would have the advantage of keeping the code together.  Even if it *was* a magic number it would be better done at the other level - you should be able to configure channel 7 on your transmitter to disable RC overrides (some people will still have radios that only transmit 8 channels)

Reply:
This is due to the fact that only the first 8 channel are overridable. We don't somebody using override to eject himself